pep-0458.txt

```diff
@@ -942,7 +942,7 @@ in this section:
    distributions and manage keys is expected to render key signing an unused
    feature.
 
-    __ https://minilock.io/
+    __ https://github.com/kaepora/miniLock
```

Comment:
I have only corrected the link here (seems like minilock.io has a new owner). However, I suggest to update the entire reference to something like YubiKey, or another contemporary alternative. If desired I can do it as part of this PR.

Reply:
Yes, please add this too!